### PR TITLE
Adding Inhabited Instances for New Types

### LINKED ISSUE #81: Add an inhabited instance for each new type declared in model.lean
Writing an inhabited instance for type **A** means proving that there is at least one term **(a : A)**, i.e. the type is nonempty.
Inhabited instances are explained in TPIL here: https://leanprover.github.io/theorem_proving_in_lean/type_classes.html#type-classes-and-instances

Currently, the following types are missing inhabited instances.

- [x] substruc
- [x] formula
- [x] sentence
- [x] theory
- [x] complete_theory

### PR DESCRIPTION
**Pull request by andrew-yin**
Defined an inhabited instance for a formula and created an outline for an inhabited instance of a sentence

### CODE REVIEW

**Comment by andrew-yin** [comment]
Currently am working on this issue, but was hoping to see if the little that I have so far is on the right track

**Comment by andrew-yin** [comment]
I added inhabited types for sentences and substructures, but I'm not positive that the implementation is fully correct. I've also made comments on what an inhabited theory could be, but I'm not sure how to write it in Lean.

File: src/model.lean [hunk 1]
```diff
@@ -714,6 +726,9 @@ a model of `T` and write `M ⊨ T` if `M ⊨ φ` for all sentences `φ ∈ T`.-/
 def theory (L : lang) : Type := set (sentence L)
 instance theory.has_mem : has_mem (sentence L) (theory L) := ⟨set.mem⟩
 
+/- A theory that is guaranteed to exist is the set {⊤'}, since ⊤' is guaranteed to be a sentence -/
```
Comment (vaibhavkarve) [inline src/model.lean:727]: You can use fconstructor or ⟨...⟩ to start writing the default theory.

**Comment by vaibhavkarve** [comment]
I reviewed and added some comments for how to proceed with `theory.inhabited`.

**Comment by andrew-yin** [comment]
Complete theory doesn't need an inhabited instance after changing implementation to a definition, model doesn't need inhabited since it's not guaranteed to exist for every theory

**Pull request merged**